pom.xml

```diff
@@ -127,10 +127,11 @@
       <artifactId>jackson-dataformat-yaml</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.kotcrab.remark/remark -->
     <dependency>
-      <groupId>com.overzealous</groupId>
+      <groupId>com.kotcrab.remark</groupId>
       <artifactId>remark</artifactId>
-      <version>${remark.version}</version>
```

Comment:
May be it's reasonable to use existing constant remark.version (with 1.2.0 value) or remove it from properties